### DISCUSSION TITLE
Is there a way to add a file type in "Restrict file types" list for the "Attachement Field"?

### DISCUSSION THREAD

**Discussion by sanderquirynen** [sulu/SuluFormBundle#281] [Q&A] [answered]
Is there a way to add a file type in "Restrict file types" list for the "Attachement Field"?
If for example you would like to limit attachments to types like `.pdf, .docx, .xlxs`

**Comment by sanderquirynen** [comment, marked as answer]
Answer from @alexander-schranz on Slack channel:

The restriction is currently based on exist types of sulu media (video, audio, image, documents). 

**Solution 1**
You could workaround your problem by validating the type in the frontend by adding in your custom form theme <input accept="my_mime_type" to the file upload input.

**Solution 2:**
You could also decorate the exist AttachmentType services: https://github.com/sulu/SuluFormBundle/blob/2.x/Dynamic/Types/AttachmentType.php

**Reply by niklasnatter** [reply]
Thanks a lot for making the answer discoverable for others! 🙂
